docs/03-apis/api-admin/groups.md

```diff
@@ -51,7 +51,10 @@ License along with DSP. If not, see <http://www.gnu.org/licenses/>.
 ```json
     {
       "name": "NewGroup",
-      "description": "NewGroupDescription",
+      "descriptions": [
+        {"value": "NewGroupDescription", "language": "en"},
+        {"value": "NeueGruppenBeschreibung", "language": "de"}
+      ],
```

Comment:
Just a comment: In fact this is a breaking change. But groups endpoint is not used yet (at least in dsp-app) but already implemented in js-lib.